.ansible-lint

```diff
@@ -14,10 +14,11 @@ exclude_paths:
   - requirements-podman.yml
 
 kinds:
-  - playbook: '**/tests/**/test_*.yml'
-  - playbook: '**/playbooks/**/*.yml'
   - tasks: '**/tasks_*.yml'
   - tasks: '**/env_*.yml'
+  - tasks: '**/tests/azure/**/*.yml'
```

Comment:
This line is good as long as ansible-lint does not enforce for example `ansible.builtin.script` use for the Azure templates.

Reply:
ok... this didn't work. I can only reproduce in Github workflow.

Added some more fine-grained control, but `tests/azure` is still being overriden, even if they are on the `exclude_paths`.